create_appdata and buildservice are not set to True after calling
setup.py externally ("sdist" and/or "install" are not used), leading to
net.displaycal.DisplayCAL.appdata.xml not being found for copying in
the build phase.

--- a/setup.py
+++ b/setup.py
@@ -639,18 +639,8 @@
         if len(sys.argv) == 1 or (len(sys.argv) == 2 and dry_run):
             return
 
-    create_appdata = (
-        (appdata or "install" in sys.argv[1:] or "sdist" in sys.argv[1:])
-        and not help
-        and not dry_run
-    )
-
-    if (
-        "sdist" in sys.argv[1:]
-        or "install" in sys.argv[1:]
-        or "bdist_deb" in sys.argv[1:]
-    ) and not help:
-        buildservice = True
+    create_appdata = True
+    buildservice = True
 
     if create_appdata or buildservice:
         with codecs.open(str(Path(pydir, "CHANGES.html")), "r", "UTF-8") as f:
